packages/react/src/elements/components/ToastList/toastList.scss

```diff
@@ -0,0 +1,31 @@
+.hig__toast-list-wrapper {
+  position: fixed;
+  right: 10px;
+}
+
+.hig__toast-list-wrapper--position-top {
+  top: 10px;
+}
+
+.hig__toast-list-wrapper--position-bottom {
+  bottom: 10px;
+}
+
+.hig__toast-list {
+  @import "../Toast/variables.scss";
```

Comment:
I assume this keeps the variable declarations from leaking into the global scope? I havn't seen this before. Cool.

Reply:
First saw it in https://github.com/Autodesk/hig/pull/802. Thanks, @morrisallison.